mars_troughs/trough.py

```diff
@@ -172,22 +169,21 @@ def get_lag_at_t(self, time):  # Model dependent
             a, b = p[0:2]  # lag(t) = a + b*t
             return a + b * time
         return  # error, since no number is returned
-
-    def get_Rt_model(self, lags, times):
+    
+    def get_retreat_model_t(self, lag_t, time):
         """
-        Computes the amount of retreat of ice at the input lag and times.
+        Calculates the values of retreat of ice per time (mm/year).
+        These values are obtained by evaluating self.ret_data_spline using
+        the lag_model_t and time values.
 
         Args:
-            time (int,float,list or np.ndarray): times at which we want to
-                                                 calculate the retreat.
-            lag (int,float,list or np.ndarray): lag for which we want to
-                                                calculate
-                                                the retreat.
+            lag_t (np.ndarray): lag per time 
```

Comment:
```suggestion
            lag_t (np.ndarray): lag at time 
```

I know in other places we wrote "lag per time" but this is a bit misleading IMO, because 'per' implies there is a division (e.g. miles per gallon). I think it might be clearer if it always read 'at' instead.